dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -1245,10 +1253,7 @@ def clean_old_provider_artifacts(
             # Leave only last version from each type
             for versioned_file in package_types[:-1]:
                 command = ["svn", "rm", versioned_file.base + versioned_file.version + versioned_file.suffix]
-                if not execute:
-                    get_console().print(f"[info]Running command: {command} in dry run[\]")
-                else:
-                    subprocess.run(command, check=True)
+                subprocess.run(command, check=True)
```

Comment:
The `--dry-run` will only be used automatically here if you use `run_command` from utils. The `subprocess_run` is stdlib one and in Breeze we have `airflow_breeze.utils.run_command` that handles all the "bells-and-whistles" :

* Support for --dry-run passed in the command (automatically)
* Support for --verbose passed in the command (automatically)
* Support for output redirection for parallel commands (not used here)
* Support for groupping command output in CI in a "Foldable" run command

Reply:
Oh yes. I refactored this portion and missed that entirely!

Reply:
Still `run()` is  used here ?

Reply:
Handled it now and pushed it as well